zoo/box2d/bipedalwalker/envs/bipedalwalker_cont_disc_env.py

```diff
@@ -13,15 +11,16 @@
 from ding.utils import ENV_REGISTRY
 from easydict import EasyDict
 
-from zoo.box2d.lunarlander.envs.lunarlander_env import BipedalWalkerEnv
+from zoo.box2d.lunarlander.envs.lunarlander_env import LunarLanderEnv
+
 
 @ENV_REGISTRY.register('bipedalwalker_cont_disc')
-class BipedalWalkerDiscEnv(BipedalWalkerEnv):
+class BipedalWalkerDiscEnv(LunarLanderEnv):
```

Comment:
Is this on purpose?

Reply:
Hello, this is a mistake, which has been corrected in the latest [commit](https://github.com/opendilab/LightZero/commit/19588617663fb196a1c64502e3f6feed12e58627). Thank you! Best wishes!

Reply:
You're welcome! Thanks for the great software. I am learning a lot about RL